.github/workflows/release.yml

```
@@ -52,26 +52,19 @@ jobs:
         name: Install signore
         uses: hashicorp/setup-signore-package@v1
       -
-        uses: hashicorp/setup-hc-releases@v1
+        name: Setup hc-releases
+        uses: hashicorp/actions-setup-hc-releases@v1
         with:
           github-token: ${{ secrets.CODESIGN_GITHUB_TOKEN }}
-      -
-        name: Configure AWS Credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.TERRAFORM_PROD_AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.TERRAFORM_PROD_AWS_SECRET_ACCESS_KEY }}
-          aws-region: us-east-1
-          role-to-assume: ${{ secrets.TERRAFORM_PROD_AWS_ROLE_TO_ASSUME }}
-          role-skip-session-tagging: true
-          role-duration-seconds: 3600
       -
         name: Release
         uses: goreleaser/goreleaser-action@v2
         with:
           version: latest
           args: release
         env:
+          HC_RELEASES_HOST_STAGING: ${{ secrets.HC_RELEASES_HOST_STAGING }}
+          HC_RELEASES_KEY_STAGING: ${{ secrets.HC_RELEASES_KEY_STAGING }}
```

The env vars should use their real names so that goreleaser configs are not hardcoded to specific environments.
```suggestion
          HC_RELEASES_HOST: ${{ secrets.HC_RELEASES_HOST_STAGING }}
          HC_RELEASES_KEY: ${{ secrets.HC_RELEASES_KEY_STAGING }}
```